inception-project/inception
# Feature/1061 switch from j cas to cas

**What's in the PR**
- Avoid use of JCas (except in test and import/export code)
- Adjust to upstream changes

**How to test manually**
* Make sure you have use build of https://github.com/webanno/webanno/pull/1262 to test this 
* Just use the application and basically try out anything imaginable. The change was pervasive.

**Automatic testing**

**Documentation**
